contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -185,6 +185,8 @@ NXmpes(NXobject):
         work_function(NX_FLOAT):
           unit: NX_ENERGY
           exists: recommended
+        voltage_energy_range(NX_FLOAT):
+          exists: optional
```

Comment:
If it's optional we don't have to state it here explicitly (sorry, that slipped in the previous PR).

Reply:
Right, we only make things optional if the term is not explictly named in the base class right, correct? So e.g., if only the generic group is defined. Like if there is an `(NXaperture)`, but we want to have the explicitly named (optional) `entrance_slit(NXaperture)` in `NXmpes`.

I removed it from NXmpes.

Reply:
> Right, we only make things optional if the term is not explictly named in the base class right, correct?

Yes, exactly.